src/Model/Profile.php

```diff
@@ -857,23 +870,34 @@ public static function openWebAuthInit($token)
 		Logger::info('OpenWebAuth: auth success from ' . $visitor['addr']);
 	}
 
-	public static function zrl($s, $force = false)
+	/**
+	 * Returns URL with URL-encoded zrl parameter
+	 *
+	 * @param string $url   URL to enhance
+	 * @param bool   $force Either to force adding zrl parameter
+	 *
+	 * @return string URL with 'zrl' parameter or original URL in case of no Friendica profile URL
+	 */
+	public static function zrl(string $url, bool $force = false)
```

Comment:
Why not add the return value type-hint?
```suggestion
	public static function zrl(string $url, bool $force = false): string
```